kclvm/config/src/vfs.rs

```diff
@@ -38,6 +41,20 @@ pub fn fix_import_path(root: &str, filepath: &str, import_path: &str) -> String
     // abspath: import path.to.sub
     // fix_import_path(root, "path/to/app/file.k", "path.to.sub") => path.to.sub
 
+    // Use canonicalize to convert the path to absolute path
+    // avoid the symbolic link issue caused the file not found
+    let filepath = fs::canonicalize(filepath)
```

Comment:
Is need `adjust_canonicalize()`  to handle Windows drive letter prefixes?

Reply:
fixed